geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/repository/IndexRepositoryImpl.java

```diff
@@ -90,8 +97,14 @@ public void create(Object key, Object value) throws IOException {
   @Override
   public void update(Object key, Object value) throws IOException {
     long start = stats.startUpdate();
+    Collection<Document> docs = Collections.emptyList();
```

Comment:
Hmm, so this will now delete the old documents mapped to this key if there is an exception? Maybe we just shouldn't call the writer if the serializer throws an exception?